.gitignore

```diff
@@ -7,3 +7,6 @@
 .venv/
 .pytest_cache/
 **__pycache__/
+db.sqlite3
+identifier.sqlite
```

Comment:
Similar to `db.sqlite3`, the `identifier.sqlite` file is also being ignored. Verify that this file does not contain any crucial data that should be version-controlled.